lib/pbench/cli/server/shell.py

```diff
@@ -182,21 +181,21 @@ def main():
     try:
         server_config = get_server_config()
     except (ConfigFileNotSpecified, BadConfig) as e:
-        print(e)
+        print(e, file=sys.stderr)
         sys.exit(1)
     logger = get_pbench_logger(PROG, server_config)
     if site.ENABLE_USER_SITE:
         find_the_unicorn(logger)
     try:
-        host = server_config.get("pbench-server", "bind_host")
-        port = str(server_config.get("pbench-server", "bind_port"))
-        db_uri = server_config.get("database", "uri")
-        db_wait_timeout = int(server_config.get("database", "wait_timeout"))
-        workers = str(server_config.get("pbench-server", "workers"))
-        worker_timeout = str(server_config.get("pbench-server", "worker_timeout"))
-        oidc_server = server_config.get("authentication", "server_url")
-        crontab_dir = server_config.get("pbench-server", "crontab-dir")
-    except (NoOptionError, NoSectionError) as exc:
+        host = server_config._get_conf("pbench-server", "bind_host")
```

Comment:
If we're going to call `_get_conf()` from outside the `pbench.server` module (i.e., outside the `PbenchServerConfig` class), shouldn't we remove the leading underscore?

Reply:
See what you think of the proposed solution.